samplestore/src/main/java/com/stripe/samplestore/PaymentActivity.java

```diff
@@ -187,28 +170,21 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
 
     private void updateConfirmPaymentButton() {
-        Long price = mCartManager.getTotalPrice();
+        long price = mStoreCart.getTotalPrice();
 
-        if (price != null) {
-            mConfirmPaymentButton.setText(String.format(Locale.ENGLISH,
-                    "Pay %s", StoreUtils.getPriceString(price, null)));
-        }
+        mConfirmPaymentButton.setText(String.format(Locale.ENGLISH,
+                "Pay %s", StoreUtils.getPriceString(price, null)));
     }
 
     private void addCartItems() {
         mCartItemLayout.removeAllViewsInLayout();
-        String currencySymbol = AndroidPayConfiguration.getInstance()
-                .getCurrency().getSymbol(Locale.US);
+        String currencySymbol = mStoreCart.getCurrency().getSymbol(Locale.US);
 
-        Collection<LineItem> items = mCartManager.getLineItemsRegular().values();
-        addLineItems(currencySymbol, items.toArray(new LineItem[items.size()]));
+        addLineItems(currencySymbol, mStoreCart.getLineItems()
+                .toArray(new StoreLineItem[mStoreCart.getSize()]));
 
-        items = mCartManager.getLineItemsShipping().values();
-        addLineItems(currencySymbol, items.toArray(new LineItem[items.size()]));
-
-        if (mCartManager.getLineItemTax() != null) {
-            addLineItems(currencySymbol, mCartManager.getLineItemTax());
-        }
+        addLineItems(currencySymbol,
```

Comment:
This isn't part of the Cart anymore, because we aren't using Google's version of a Cart.